binaries/cuprated/Cargo.toml

```diff
@@ -8,6 +8,68 @@ authors = ["Boog900", "hinto-janai", "SyntheticBird45"]
 repository = "https://github.com/Cuprate/cuprate/tree/main/binaries/cuprated"
 
 [dependencies]
+# TODO: after v1.0.0, remove unneeded dependencies.
```

Comment:
Added all the workspace dependencies to `cuprated` for now for convenience. We can remove them after v1.0.0.